04-Intro_to_Vue/assignment/my-app-name/src/components/GridSystem.vue

```diff
@@ -0,0 +1,29 @@
+<script setup>
+import { ref } from "vue";
+const navLinks = ref(["Home", "Features", "Pricing", "About"]);
+</script>
+<template>
+  <main class="main">
+    <div class="container mt-3">
+      <div class="row">
+        <!-- Menu Section -->
+        <nav class="col-xl-3 col-lg-4 col-md-5 col-sm-12 bg-light col-xs-3">
```

Comment:
it's good to keep your classes tidy. I'd move the bg-light either to the end or beginning of the list of classes